<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Todos</class>
 <widget class="QWidget" name="Todos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1420</width>
    <height>781</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Montserrat Medium</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1437</width>
     <height>569</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="ComponentLayout">
    <property name="spacing">
     <number>30</number>
    </property>
    <property name="leftMargin">
     <number>120</number>
    </property>
    <property name="topMargin">
     <number>120</number>
    </property>
    <property name="rightMargin">
     <number>120</number>
    </property>
    <property name="bottomMargin">
     <number>100</number>
    </property>
    <item>
     <widget class="QLabel" name="TodosHeader">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Montserrat</family>
        <pointsize>30</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
color: rgb(143, 115, 101);
}</string>
      </property>
      <property name="text">
       <string>Today's Plan</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="TodosLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="EmptyLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Montserrat Medium</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
color: rgb(129,129,129)
}</string>
        </property>
        <property name="text">
         <string>No plans for today</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="NewTodoBtn">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Montserrat Medium</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
color: rgb(31,31,31);
background-color: rgb(143,115,101);
border: none;
border-radius: 10px;
}
QPushButton:hover {
background-color: rgb(176,153,141);
}</string>
      </property>
      <property name="text">
       <string>New Todo</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="NewTodoForm">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QLineEdit" name="NameInput">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>815</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>815</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Montserrat Medium</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
color: rgb(220,220,220);
background-color: rgb(41,41,41);
border: none;
border-radius: 10px;
padding: 0 15px;
}</string>
        </property>
        <property name="placeholderText">
         <string>Get a cup of coffee</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="StartInput">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Montserrat Medium</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
color: rgb(220,220,220);
background-color: rgb(41,41,41);
border: none;
border-radius: 10px;
padding: 0 15px;
}</string>
        </property>
        <property name="placeholderText">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="EndInput">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Montserrat Medium</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
color: rgb(220,220,220);
background-color: rgb(41,41,41);
border: none;
border-radius: 10px;
padding: 0 15px;
}</string>
        </property>
        <property name="placeholderText">
         <string>End</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SaveBtn">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Montserrat Medium</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
color: rgb(31,31,31);
background-color: rgb(143,115,101);
border: none;
border-radius: 10px;
}
QPushButton:hover {
background-color: rgb(176,153,141);
}</string>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="InvalidMsg">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Montserrat</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
color: rgb(220,220,220);
background-color: rgb(231,115,115);
border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>Unknown error occured, cannot add todo</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
